### DISCUSSION TITLE
Can't install latest version on Ubuntu

### DISCUSSION THREAD

**Discussion by Fabeuss** [dundee/gdu#18] [Q&A] [answered]
Sorry but I can't find the solution to install the latest versions on Ubuntu. 
The latest deb (gdu_4.0.0-1_amd64.deb) is 4.00...
Could you please explain me how to do that ?

**Comment by dundee** [comment, marked as answer]
You are right. I removed the `debian` packaging from gdu because I am trying to do it the "proper" way, get `gdu` directly to Debian archive. But it will take some time (years :rofl:), so I will upload custom `deb` packages till then again.

**Reply by Fabeuss** [reply]
Ok, thanks for your answer 😊 Not years but I will wait...😉

**Comment by dundee** [comment]
Added to https://github.com/dundee/gdu/releases/tag/v4.2.0 

Enjoy!

**Reply by Fabeuss** [reply]
Wow, that was faster than expected, you're a god 👍😁
